File: examples/pascal/constructs/for_loop_continue.pas
// Ported from examples/lumen/constructs/for_loop_continue.lm by scripts/port_examples.py; edit the Lumen original, not this file.
var i: integer;

begin
    i := 0;
    while i < 11 do begin
        if i = 5 then begin
            i := i + 1;
            continue;
        end;
        write(i);
        if i < 10 then begin
            write(', ');
        end;
        i := i + 1;
    end;
    writeln('');
end.

File: examples/php/constructs/for_loop_continue.php
<?php
// Ported from examples/lumen/constructs/for_loop_continue.lm by scripts/port_examples.py; edit the Lumen original, not this file.
$i = 0;
while ($i < 11) {
    if ($i == 5) {
        $i = $i + 1;
        continue;
    }
    print($i);
    if ($i < 10) {
        print(", ");
    }
    $i = $i + 1;
}
print("\n");
